vulnerabilities/dos-revert.md

```diff
@@ -83,7 +83,7 @@ Regardless of usage of checked math, it's necessary to ensure that any valid inp
 
 ### Unexpected Balance
 
-It's important to take caution in enforcing expected contract balances of tokens or Ether as those balances may be increased by an attacker to cause an unexpected revert. This is easily possible with ERC20 tokens by simply `transfer`ring to the contract, but is also possible with Ether by [Forcibly sending Ether to a contract](./forcibly-sending-ether.md).
+It's important to take caution in enforcing expected contract balances of tokens or Ether as those balances may be increased by an attacker to cause an unexpected revert. This is easily possible with ERC20 tokens by simply `transfer`ring to the contract, but is also possible with Ether by forcibly sending Ether to a contract.
```

Comment:
@0xSandyy, thank you for your work on this PR! I noticed that you mentioned deleting the reference to ./forcibly-sending-ether.md in dos-revert.md since it has not been implemented yet. Instead of removing the reference, I suggest updating it to Force-Feeding (force-feeding.md) as this will be added from PR #63.

Reply:
Actually, I've suggested the force-feeding vuln to be added to this file under this section. See: https://github.com/kadenzipfel/smart-contract-vulnerabilities/pull/63#issuecomment-2148541319. But let's keep this open until that PR gets merged